File: src/main/java/itmo/is/project/mapper/module/dock/DockModuleBlueprintMapper.java
package itmo.is.project.mapper.module.dock;

import itmo.is.project.dto.module.dock.DockModuleBlueprintDto;
import itmo.is.project.mapper.EntityMapper;
import itmo.is.project.model.module.dock.DockModuleBlueprint;
import org.mapstruct.Mapper;

@Mapper
public interface DockModuleBlueprintMapper extends EntityMapper<DockModuleBlueprintDto, DockModuleBlueprint> {
}

File: src/main/kotlin/itmo/isproject/dock/mapper/DockModuleBlueprintMapper.kt
package itmo.isproject.dock.mapper

import itmo.isproject.dock.dto.DockModuleBlueprintDto
import itmo.isproject.shared.common.mapper.EntityMapper
import itmo.isproject.dock.model.DockModuleBlueprint
import org.mapstruct.Mapper

@Mapper
interface DockModuleBlueprintMapper : EntityMapper<DockModuleBlueprintDto, DockModuleBlueprint>
